### ISSUE TITLE
Why does the loss of dice stay the same?

### ISSUE THREAD

**Issue by Geekiter** [kingo233/FCT-Pytorch#1]
```
2023-03-09 12:21:47,350 |     INFO |    main.py:311 | -----------------------------------
2023-03-09 12:21:47,350 |     INFO |    main.py:312 | epoch: 197,train loss:0.091478,validate loss:0.091043,dice:0.256989,LV dice:0.008357,RV dice:0.030641,MYO dice:0.008357
2023-03-09 12:21:47,350 |     INFO |    main.py:315 | -----------------------------------
2023-03-09 12:25:06,750 |     INFO |    main.py:311 | -----------------------------------
2023-03-09 12:25:06,750 |     INFO |    main.py:312 | epoch: 198,train loss:0.091478,validate loss:0.091043,dice:0.256989,LV dice:0.008357,RV dice:0.030641,MYO dice:0.008357
2023-03-09 12:25:06,750 |     INFO |    main.py:315 | -----------------------------------
2023-03-09 12:28:26,582 |     INFO |    main.py:311 | -----------------------------------
2023-03-09 12:28:26,582 |     INFO |    main.py:312 | epoch: 199,train loss:0.091478,validate loss:0.091043,dice:0.256989,LV dice:0.008357,RV dice:0.030641,MYO dice:0.008357
2023-03-09 12:28:26,582 |     INFO |    main.py:315 | -----------------------------------
```

**Comment by kingo233** [comment]
Hi! Thank you for your attention. I have noticed this problem.I guess it have something to do with model's code. Based on the visualized results from TensorBoard, it seems that there is a gradient vanishing issue in the network. I am currently working on resolving it

**Comment by kingo233** [comment]
Hi! the problem of dice stay same have been solved.Enjoy it!
